pkgs/development/python-modules/sortedcontainers/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "sortedcontainers";
+  version = "2.0.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "607294c6e291a270948420f7ffa1fb3ed47384a4c08db6d1e9c92d08a6981982";
+  };
+
+  # tries to run tests for all python versions and uses virtualenv weirdly
+  doCheck = false;
+  #buildInputs = with self; [ tox nose ];
```

Comment:
That's not true.
```
checkPhase = ''
  py.test
''
```
should suffice.

Reply:
Those lines were left over from the original. The Pypi tarball does not come with tests. So I have now set `doCheck = false;` with appropriate comment.